libarchive/test/test_read_pax_header_kw.c

```diff
@@ -28,8 +28,12 @@
  */
 
 #include "test.h"
+// /app/bin/libarchive_test -vv -r /app/libarchive/test test_write_pax_kw
```

Comment:
does upstream care about the use of C++ style comments?

Reply:
There are other files with C++ comments, so going to run with it... if & when upstream, we can fix too. I'm guessing more unit tests could be added too, only covering basics right now.